Formula/pygobject3.rb

```diff
@@ -16,22 +15,32 @@ class Pygobject3 < Formula
   option "without-python", "Build without python3 support"
   option "with-python@2", "Build with python2 support"
 
-  depends_on "pkg-config" => :build
-  depends_on "libffi" => :optional
   depends_on "glib"
+  depends_on "gobject-introspection"
+  depends_on "libffi" => :optional
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
   depends_on "python@2" => :optional
-  depends_on "python" => :recommended
+  depends_on "python" => [:build, :recommended]
   depends_on "py2cairo" if build.with? "python@2"
   depends_on "py3cairo" if build.with? "python"
-  depends_on "gobject-introspection"
 
   def install
-    Language::Python.each_python(build) do |python, _version|
-      system "./configure", "--disable-dependency-tracking",
-                            "--prefix=#{prefix}",
-                            "PYTHON=#{python}"
-      system "make", "install"
-      system "make", "clean"
+    Language::Python.each_python(build) do |python, version|
+      mkdir "build#{version}" do
+        system "meson", "--prefix=#{prefix}",
+                        "-Dpycairo=true",
+                        "-Dpython=#{python}",
+                        ".."
+        # avoid linking against python framework
```

Comment:
This should maybe be reported upstream, if this is the issue I think it is.

Reply:
Done at https://gitlab.gnome.org/GNOME/pygobject/issues/253